site/content/en/docs/Installation/Install Agones/yaml.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Install Agones using YAML"
+linkTitle: "YAML"
+weight: 10
+description: >
+  We can install Agones to the cluster using an install.yaml file.
+---
+
+```bash
+kubectl create namespace agones-system
+kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/install/yaml/install.yaml
+```
+
+You can also find the install.yaml in the latest `agones-install` zip from the [releases](https://github.com/googleforgames/agones/releases) archive.
+
+{{< alert title="Warning" color="warning">}}
+Installing Agones with the `install.yaml` will setup the TLS certificates stored in this repository for securing
+kubernetes webhooks communication. If you want to generate new certificates or use your own,
+we recommend using the helm installation.
+{{< /alert >}}
+
+## Confirm Agones is running
```

Comment:
Maybe say "Next Step" here (like is done in the create cluster sections) since the body of the section explains what to do?